sources/osg/Utils.js

```diff
@@ -177,18 +177,17 @@ var times = {};
 
 // we bind the function to Notify.console once and for all to avoid costly apply function
 
-Utils.time = ( Notify.console.time || function ( name ) {
+Utils.time = ( function ( name ) {
     times[ name ] = Timer.instance().tick();
 } ).bind( Notify.console );
 
-Utils.timeEnd = ( Notify.console.timeEnd || function ( name ) {
-
+Utils.timeEnd = ( function ( name ) {
     if ( times[ name ] === undefined )
         return;
 
     var duration = Timer.instance().deltaM( times[ name ], Timer.instance().tick() );
 
-    Notify.debug( name + ': ' + duration + 'ms' );
+    Notify.info( name + ': ' + duration + 'ms' );
```

Comment:
Changed back to debug. Will change behavior compared to using console.timeEnd like before that was always printed, but more correct to put this at a debug level